src/collar/oc_settings.lsl

```diff
@@ -393,7 +398,7 @@ default {
             g_lSettings = SetSetting(g_lSettings, sToken, sValue);
             if (sToken == "intern_news") {
                 g_fLastNewsStamp = (float)sValue;
-                g_kURLRequestID = llHTTPRequest(g_sEmergencyURL+"attn.txt",[HTTP_METHOD,"GET",HTTP_VERBOSE_THROTTLE,FALSE],"");
+                g_kURLRequestID = llHTTPRequest(g_sEmergencyURL+"attn"+HTTP_TYPE,[HTTP_METHOD,"GET",HTTP_VERBOSE_THROTTLE,FALSE],"");
```

Comment:
I got rid of these HTTP_TYPE variables in a few scripts recently.  I'd like to not bring them back.

Reply:
agreed, was trying to keep all scripts in all grids up to your latest with changes but missed a few